scripts/start

```diff
@@ -0,0 +1,3 @@
+export NODE_ENV=$1
+export PORT=$2
+node --harmony build/server.js
```

Comment:
need to make sure current directory is expected, we can use something like below

```
cd `dirname $0`/ >/dev/null 2>&1
DIR=`pwd`
echo "pwd is ${DIR}"
```